Dynamic-Programming/tests/LevenshteinDistance.test.js

```diff
@@ -0,0 +1,19 @@
+import { calculateLevenshteinDp } from '../LevenshteinDistance'
+
+test('Add and remove letters at the beginning and the end while making replacements', () => {
+  const from = 'kitten'
+  const to = 'sitting'
+  expect(calculateLevenshteinDp(from, to)).toBe(3)
+})
+
+test('Replace contiguous letters at the middle of the words', () => {
```

Comment:
Suggestion :
"Should return distance based on characters at the ends of strings"